test/test_torch.py

```diff
@@ -2335,6 +2335,15 @@ def logcumsumexp(a, axis):
                 'expected scalar_type Double but found Float'):
             torch.logcumsumexp(b, axis, out=inplace_out)
 
+        # Check dim out of range
```

Comment:
instead of adding error case here, can you please add ErrorInput to `logcumsumexp` OpInfo tests in common_methods_invocations. You'd need to add `error_inputs_func` to `logcumsumexp` entry, you can look at other examples to get an idea.

Reply:
thanks, taking a look at `common_methods_invocations`